Examples/Scotty-tvOSExample/Routing/Routable+App.swift

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
This class looks to be identical to the existing one in the iOS example. Could we create a top-level `Examples/Common` folder that we stick this in and then tell Xcode to include it in both the iOS example and tvOS example targets? That way we don't have a bunch of duplicated example code that will likely be a pain to maintain and update across all of the examples. We'd probably need to update the module name for the tvOS target first, as @wmcginty suggested below.